ci/generate_azure_pipelines_matrices.py

```diff
@@ -136,6 +158,16 @@ def __init__(self, platform, build_jobs):
             "desktop",
             "win64_msvc2019_arm64",
             "msvc2019_arm64",
+            is_autodesktop=True,  # Should install win64_msvc2019_arm64 in parallel
+        ),
+        BuildJob(
+            "install-qt",
+            "6.7.3",
+            "windows",
+            "desktop",
+            "win64_llvm_mingw",
+            "llvm-mingw_64",
+            mingw_variant="win64_llvm_mingw1706",
             is_autodesktop=True,    # Should install win64_msvc2019_arm64 in parallel
```

Comment:
Is this correct? Does the llvm toolchain really need the msvc toolchain in parallel?

Reply:
Good catch. The autodesktop flag is for automatically installing the desktop architecture of Qt in parallel with an android or wasm architecture of Qt. I think the flag will be ignored in this context. Likely a copy-paste error, but benign.